bin/run_functions.js

```diff
@@ -67,11 +69,17 @@ function conflictingOptionsCheck(options) {
     if (('securitytest' in options) && (('testcase' in options) || ('testsuite' in options))) {
         util.printErrorAndExit('Error: Parameters testsuite and testcase are not allowed when securitytest is used');
     }
+
     if ('tags' in options) {
         if (('testsuite' in options) || ('testcase' in options)) {
-            util.printErrorAndExit('Error: tags cannot be used together with testcase/testsuite ');
+            util.printErrorAndExit('Error: tags cannot be used together with testcase/testsuite');
         }
     }
+
+    if(('endpoint' in options) && ('environment' in options)) {
+        util.printErrorAndExit('Error: environment cannot be used together with endpoint');
```

Comment:
**endpoint** should be called hostAndPort, the way it is in our API

Reply:
All occurrences of endpoint have been replaced with hostAndPort.